docs/gestor-tareas.md

```diff
@@ -0,0 +1,37 @@
+# Gestor de tareas
+
+Para el lenguaje de programación de este proyecto, Go, existen varías opciones conocias para la gestión de tareas, de las cuales se destacan las siguientes:
```

Comment:
Ya os dije en clase repetidamente que teníais que establecer de antemano cuales son las necesidades. Poner una serie de herramientas y elegir una aleatoriamente no cubre este objetivo, donde se trata de que sepáis tomar y documentar una elección técnica. ¿Te parece que si no pone para qué quieres algo y el contexto de la elección lo estás documentando correctamente?